fitbenchmarking/fitbenchmarking
# Make the links relative

#### Description of Work

Fixes #615 

I've also made performance profile links relative, and added a sanitiser for the minimizer names (":" and " " were breaking things for me on Windows)

#### Testing Instructions

1. run fitbenchmarking and check all outputs render correctly
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
